docs/articles/documentation/guides/advanced-guides/test-https-features-and-http2-websites.md

```diff
@@ -61,6 +137,16 @@ await runner
     .run();
 ```
 
+### Use a Trusted Certificate
```

Comment:
Please move this section before the section about self-signed certificates (because it's preferred).